Changelog.md

```diff
@@ -5,6 +5,7 @@ Language Features:
 
 Compiler Features:
  * EVM: Support for the EVM Version "Cancun".
+ * Inline Assembly: Support for transient storage opcodes ``tstore`` and ``tload``.
```

Comment:
This should go under language features and for `Yul` in general, not just `Inline assembly`.

This would be more consistent with how we describe the feature in other PRs:
```suggestion
 * Yul: Introduce builtins ``tload()`` and ``tstore()`` for transient storage access.
```

Reply:
It's still in the "Compiler Features" section. Should be "Language Features".